lib/huiClient.js

```diff
@@ -0,0 +1,47 @@
+const log = require('@financial-times/n-logger').default;
+
+const huiBasePath = process.env.HUI_BASE_PATH;
+const licencePath = '/licences/';
+const options = {
+  method: 'GET',
+  headers: {
+    'X-Api-Key': process.env.HUI_API_KEY
+  }
+};
+
+function getUsage(licenceId, aggregation, startDate, endDate, type = '', filter = '') {
+  let start = '';
+  let end = '';
+  let url = `${huiBasePath}${licencePath}${licenceId}?aggregation=${aggregation}`;
+
+  // Get the prev week/month data too. We make a clone of the start date because
+  // moments are mutable.
+  const adjustedStartDate = startDate.clone().startOf(aggregation === 'week' ? 'isoweek' : aggregation).subtract(1, aggregation);
+
+  const dateFormat = aggregation === 'week' ? '[wk]W/GGGG' : '[m]M/GGGG';
```

Comment:
This looks good but, previously, the week start/end dates were created based on isoWeek (and year). Not sure if this results in the same dates.
https://github.com/Financial-Times/kat-usage-report/blob/master/server/lib/hui.js#L66